kubeutil.py

```diff
@@ -224,6 +227,44 @@ def get_namespace(self, namespace: str):
             return ns
         return None
 
+class KubernetesServiceHelper(object):
+    '''Helper class for kubernetes service objects'''
+
+    def __init__(self, core_api: CoreV1Api):
+        self.core_api = core_api
+
+    def replace_or_create_service(self, namespace: str, service: V1Service):
+        '''Create a service in a given namespace. If the service already exists,
+        the previous version will be deleted beforehand
+        '''
+        ensure_not_empty(namespace)
+        ensure_not_none(service)
+
+        service_name = service.metadata.name
+        existing_service = self.get_service(namespace=namespace, name=service_name)
+        if existing_service:
+            self.core_api.delete_namespaced_service(namespace=namespace, name=service_name)
+        self.create_service(namespace=namespace, service=service)
+
+    def create_service(self, namespace: str, service: V1Service):
+        '''Create a service in a given namespace.'''
+        ensure_not_empty(namespace)
+        ensure_not_none(service)
+        self.core_api.create_namespaced_service(namespace=namespace, body=service)
+
+    def get_service(self, namespace: str, name: str) -> V1Service:
+        '''Return the `V1Service` with the given name in the given namespace, or `None`'''
```

Comment:
..if not such service exists (specify the condition upon which None is returned; also, optionally: use @return?